src/components/heading/heading.js

```diff
@@ -128,11 +131,17 @@ class Heading extends React.Component {
   get back() {
     if (!this.props.backLink) { return null; }
 
+    let props = { href: this.props.backLink };
```

Comment:
I don't think we should send props of backLink to the Link element as it doesn't do anything with this. I would supply the link with a onClick or a href or nothing. Open to debate though @Sage/carbon